CASMTRIAGE-4044 (#2406)
Cray-HPE/docs-csm: upgrade/Stage_0_Prerequisites.md

   git push
   ```

## Stage 0.3 - Customize the new NCN image and update NCN personalization configurations

There are two possible scenarios. Follow the procedure for the scenario that is applicable to the upgrade being performed.

- [Standard upgrade](#standard-upgrade) - Upgrading CSM on a system that has products installed other than CSM.
- [CSM-only system upgrade](#csm-only-system-upgrade) - Upgrading CSM only on a CSM-only system **no other products installed or being upgraded**.

> Note: You will not be able to rebuild NCNs on the current, pre-upgraded CSM version after performing these steps. Rebuilding NCNs will become the same thing as upgrading them.

Reviewer: ```suggestion
**NOTE:** You will not be able to rebuild NCNs on the current, pre-upgraded CSM version after performing these steps. Rebuilding NCNs will become the same thing as upgrading them.
```

> **NOTE:** You will not be able to rebuild NCNs on the current, pre-upgraded CSM version after performing these steps. Rebuilding NCNs will become the same thing as upgrading them.

Reviewer 2: ```suggestion
**NOTE:** It will not be possible to rebuild NCNs on the current, pre-upgraded CSM version after performing these steps. Rebuilding NCNs will become the same thing as upgrading them.
```

> **NOTE:** It will not be possible to rebuild NCNs on the current, pre-upgraded CSM version after performing these steps. Rebuilding NCNs will become the same thing as upgrading them.

Reviewer 2: Isn't this note only true for the standard upgrade path?

Reviewer 3: > I made one linting commit to this PR, and depending on the answer to my question, an additional change will be needed. That will need to also get added to the backport.

I undid the backport, will re-backport after this is settled.

Author: addressed
